rest/route/aws_test.go

```diff
@@ -67,6 +66,7 @@ func TestAWSSNSNotificationHandlers(t *testing.T) {
 		Id:        "agent_host",
 		StartTime: time.Date(2009, time.November, 10, 23, 0, 0, 0, time.UTC),
 		StartedBy: evergreen.User,
+		Provider:  evergreen.ProviderNameMock,
```

Comment:
is this necessary?

Reply:
`units/host_termination` emits a "host termination for a non-spawnable distro" notice, which faked me out while I was debugging, and this prevents that from happening.